### PR TITLE
Add support for bridge mode

### PR DESCRIPTION
**Pull request by M1cha**
This simply updates `common`, as there is nothing to do from the podman side. I added documentation for this new option and improved the `--internal` documentation.

Related:
- https://github.com/containers/netavark/pull/1126
- https://github.com/containers/common/pull/2247

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
@Luap99 PTAL

File: docs/source/markdown/podman-network-create.1.md [hunk 1]
```diff
@@ -118,6 +126,9 @@ Additionally the `bridge` driver supports the following options:
 - `com.docker.network.bridge.name`: This option assigns the given name to the created Linux Bridge
 - `com.docker.network.driver.mtu`: Sets the Maximum Transmission Unit (MTU) and takes an integer value.
 - `vrf`: This option assigns a VRF to the bridge interface. It accepts the name of the VRF and defaults to none. Can only be used with the Netavark network backend.
+- `mode`: This option sets the specified bridge mode on the interface. Defaults to `managed`. Supported values:
+  - `managed`: Podman creates and deleted the bridge. It will also setup sysctls and firewall rules for it.
```
Comment (Luap99) [inline docs/source/markdown/podman-network-create.1.md:130]: Maybe add that the firewall rules are added to MASQUERADE/NAT the traffic and for port forwarding DNAT rules are added. And then in the unmanaged case make clear no firewall rules are added which means no DNAT rules for port forwarding as well which might surprise users if they add ports but they don't work.

Comment (M1cha) [reply docs/source/markdown/podman-network-create.1.md]: good idea. added.

**Review by Luap99 — APPROVED** [review]
/lgtm

Thanks

**Pull request merged**